`make build docker` failed with "COPY failed: file not found in build context or excluded by .dockerignore: stat .build/linux-amd64/prometheus: file does not exist"

I'm on a Mac, I guess I'm holding it wrong, but I can't figure this one out...

```
make build docker
>> writing assets
# Un-setting GOOS and GOARCH here because the generated Go code is always the same,
# but the cached object code is incompatible between architectures and OSes (which
# breaks cross-building for different combinations on CI in the same container).
cd web/ui && GO111MODULE=on GOOS= GOARCH= go generate -x -v  -mod=vendor
doc.go
go run -mod=vendor assets_generate.go
ui.go
>> building binaries
GO111MODULE=on /Users/twilkie/go/bin/promu build --prefix /Users/twilkie/Documents/repos/prometheus 
 >   prometheus
 >   promtool
>> bundling npm licenses
rm -f "npm_licenses.tar.bz2"
find web/ui/react-app/node_modules -iname "license*" | tar cfj "npm_licenses.tar.bz2" --transform 's/^/npm_licenses\//' --files-from=-
docker build -t "prom/prometheus-linux-amd64:master" \
		-f ./Dockerfile \
		--build-arg ARCH="amd64" \
		--build-arg OS="linux" \
		./
Sending build context to Docker daemon  742.3MB
Step 1/22 : ARG ARCH="amd64"
Step 2/22 : ARG OS="linux"
Step 3/22 : FROM quay.io/prometheus/busybox-${OS}-${ARCH}:latest
 ---> 14d98f19c979
Step 4/22 : LABEL maintainer="The Prometheus Authors <prometheus-developers@googlegroups.com>"
 ---> Using cache
 ---> d323d10c7b66
Step 5/22 : ARG ARCH="amd64"
 ---> Using cache
 ---> 70d7ff8e87dd
Step 6/22 : ARG OS="linux"
 ---> Using cache
 ---> 96a2eede2dd4
Step 7/22 : COPY .build/${OS}-${ARCH}/prometheus        /bin/prometheus
COPY failed: file not found in build context or excluded by .dockerignore: stat .build/linux-amd64/prometheus: file does not exist
make: *** [common-docker-amd64] Error 1
```

--- a/.gitignore
+++ b/.gitignore
@@ -23,3 +23,4 @@ npm_licenses.tar.bz2
 /web/ui/assets_vfsdata.go
 
 /vendor
+/.build

--- a/README.md
+++ b/README.md
@@ -91,8 +91,18 @@ The Makefile provides several targets:
   * *test-short*: run the short tests
   * *format*: format the source code
   * *vet*: check the source code for common errors
-  * *docker*: build a docker container for the current `HEAD`
   * *assets*: build the new experimental React UI
+
+### Building the Docker image
+
+The `make docker` target is designed for use in our CI system.
+You can build a docker image locally with the following commands:
+
+    $ make promu
+    $ promu crossbuild -p linux/amd64
+    $ make common-docker-amd64
+
+*NB* if you are on a Mac, you will need [gnu-tar](https://formulae.brew.sh/formula/gnu-tar).
 
 ## React UI Development
 

